include/pmacc/particles/memory/boxes/ExchangePushDataBox.hpp

```diff
@@ -74,21 +81,20 @@ namespace pmacc
             DataSpace<DIM> const& superCell,
             T_Hierarchy const& hierarchy)
         {
-            TYPE oldSize
-                = alpaka::atomicAdd(worker.getAcc(), currentSizePointer, count, hierarchy); // get count VALUEs
+            TYPE oldSize = alpaka::atomicAdd(worker.getAcc(), m_particleCount, count, hierarchy); // get count VALUEs
```

Comment:
```suggestion
            // get count VALUEs
            TYPE oldSize = alpaka::atomicAdd(worker.getAcc(), m_particleCount, count, hierarchy);
```
You know... once upon a time a wise old person taught me an important lesson...

Reply:
`... old ...`? :older_man: